sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/eventsourcedentity/EventSourcedEntitiesImpl.scala

```diff
@@ -252,20 +251,21 @@ final class EventSourcedEntitiesImpl(
       }
   }
 
-  private class CommandContextImpl(
-      override val entityId: String,
-      override val sequenceNumber: Long,
-      override val commandName: String,
-      override val commandId: Long,
-      override val metadata: Metadata)
-      extends AbstractContext(system)
-      with CommandContext
-      with ActivatableContext
-
   private class EventSourcedEntityContextImpl(override final val entityId: String)
       extends AbstractContext(system)
       with EventSourcedEntityContext
   private final class EventContextImpl(entityId: String, override val sequenceNumber: Long)
       extends EventSourcedEntityContextImpl(entityId)
       with EventContext
 }
+
+class CommandContextImpl(
```

Comment:
`private[eventsourcedentity]` and `final` (if it still needs to be out here)

Reply:
not anymore, I'll put it back